swing/src/net/sf/openrocket/gui/util/GUIUtil.java

```diff
@@ -258,6 +250,7 @@ public static void setBestLAF() {
 		try {
 			// Set system L&F
 			UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+			UIManager.put("TabbedPane.foreground", Color.black);
```

Comment:
I think we should only set this in the OSX case. From SwingStartup there's a unique invocation to OSXSetup.setupOSX() when on Mac. It _might_ work just moving this into the OSXSetup.setupOSX() method, otherwise we'll want to wrap this with an if block for the platform. Also, add a comment on why we're doing this - someone will come along in the future and remove this thinking it does nothing, but a little breadcrumb about why will go a long way.